src/System.Private.CoreLib/src/System/Runtime/InteropServices/Marshal.CoreCLR.cs

```diff
@@ -713,12 +710,12 @@ public static T[] GetObjectsForNativeVariants<T>(IntPtr aSrcNativeVariant, int c
         [MethodImpl(MethodImplOptions.InternalCall)]
         public static extern int GetEndComSlot(Type t);
 
-        public static object BindToMoniker(string monikerName)
+        public static object? BindToMoniker(string monikerName)
```

Comment:
When can this return null?

Reply:
Ah, I missed that the P/Invoke methods were `PreserveSig = false`; so they will throw if any of them fail. I'll fix this.

Reply:
Fixed.